### PR TITLE
Update docs to fix incorrect tag for JS tests

### PR DESCRIPTION
**Pull request by paulsheldrake**
Fix incorrect documentation about running tests with javascript.

### CODE REVIEW

**Comment by lmakarov** [comment]
@paulsheldrake what makes you think this changes is necessary?

Mink docs still suggest `@javascript` as a tag to trigger testing via Selenium
http://docs.behat.org/en/v2.5/cookbook/behat_and_mink.html

**Comment by paulsheldrake** [comment]
I was having trouble getting my tests to run and I talked to @sean-e-dietrich about it and he mentioned that the tag should be @js and not @javascript.   once I had made that change then it worked

**Comment by lmakarov** [comment]
That's weirded.

@sean-e-dietrich we should add a full test suite for to repo to make sure the Behat stuff is also run by Travis.

File: tests/behat/behat.common.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ default:
       # Use goutte (basic PHP browser, super fast) as the default driver.
       default_session: goutte
       # For real browser testing and tests requiring JS use selenium2 driver.
-      # Tag features/scenarious with @javascript to use the selenium2 driver.
+      # Tag features/scenarious with @js to use the selenium2 driver.
```
Comment (lmakarov) [inline tests/behat/behat.common.yml]:
```suggestion
      # Tag features/scenarios with @js to use the selenium2 driver.
```

**Comment by sean-e-dietrich** [comment]
@paulsheldrake it looks like I may have been mistaken when I spoke to you and the tag is actually `@javascript`. It does look like though our test was not configured to have the selenium container to test in and maybe that is possibly what happened.

**Pull request closed without merging**